src/_scss/pages/state/_contentWrap.scss

```diff
@@ -1,33 +1,198 @@
+
+			@media(min-width: $large-screen) {
+				display: flex;
+				justify-content: end;
+			}
+
+			.state__line-div {
+				@media(min-width: $large-screen) {
+					border-right: 1px solid $color-gray-lighter;
+					line-height: 24px;
+					height: 24px;
+					margin-left: 8px;
+				}
+			}
+
+			// overrides to level out the text in the row
+			.rounded-toggle__wrapper {
+				margin-top: rem(2);
+			}
+
+			.accordion-container {
+				.accordion {
+					.heading {
+						padding-top: rem(10);
+					}
+				}
+			}
+
+			.accordion-container {
+				@media(max-width: $tablet-screen) {
+					margin-bottom: rem(4);
+				}
+				@media(min-width: $tablet-screen) {
+					margin-left: rem(10);
+				}
+			}
+
+			.accordion-container .heading {
+				height: 14px;
+				color: $color-primary;
+				border: none;
+				font-size: $font-size-14;
+				max-width: rem(120);
+				padding: 0;
+				margin: 0;
+				justify-content: unset !important;
+
+				&:hover,
+				&:active,
+				&:focus,
+				&:visited {
+					color: $color-primary-darker;
+				}
+
+				@media(min-width: $tablet-screen) {
+					padding-top: rem(8);
+				}
+			}
+
+			.accordion-container .plus {
+				width: 10px;
+				height: 10px;
+
+				&:hover,
+				&:active,
+				&:focus,
+				&:visited {
+					color: $color-primary-darker;
+				}
+			}
+
+			.accordion-container .minus {
+				width: 10px;
+				height: 10px;
+
+				&:hover,
+				&:active,
+				&:focus,
+				&:visited {
+					color: $color-primary-darker;
+				}
+			}
+
+			.accordion-container .toggle {
+				color: $color-primary;
+				font-size: $font-size-14;
+				font-weight: $font-semibold;
+				padding-left: rem(6);
+
+				&:hover,
+				&:active,
+				&:focus,
+				&:visited {
+					color: $color-primary-darker;
+				}
+
+				@media(min-width: $tablet-screen) {
+					padding-right: rem(6);
+				}
+			}
+
+			.accordion-container .open .content {
+				display: none;
+			}
+		}
+
+		.state__what-content {
+			border-left: 8px solid $color-primary-alt-light;
+			border-top: 1px solid $color-primary-alt-light;
+			border-right: 1px solid $color-primary-alt-light;
+			border-bottom: 1px solid $color-primary-alt-light;
+			background-color: #e0f3f8;
+			padding: 24px 16px;
+			margin-left: 24px;
+			margin-right: 8px;
+			margin-bottom: 16px;
+			@media(max-width: $tablet-screen) {
+				margin-left: 0;
+				margin-right: 0;
+			}
+			@media(min-width: $tablet-screen) and (max-width: $medium-screen) {
+				margin-left: 0;
+				margin-right: 0;
+				margin-top: 32px;
+			}
+			.state__what-heading {
+				font-size: rem(16);
+				font-weight: $font-semibold;
+				display: inline-block;
+				margin: 0;
+				margin-left: 18px;
+				color: $gray-90;
+			}
+	
+			.state__what-second-heading {
+				font-size: rem(16);
+				font-weight: $font-semibold;
+				display: inline-block;
+				color: $gray-90;
+				margin: 24px 16px 0 40px;
+			}
+	
+			.state__what-text {
+				margin-left: 40px;
+				font-size: rem(14);
```

Comment:
font size variable, looks like there are a few places this could be updated

Reply:
updated